### DISCUSSION TITLE
Implement a Copy to Clipboard Feature enabled to the Button Component

### DISCUSSION THREAD

**Discussion by arvind-iyer-2001** [resend/react-email#1324] [Ideas]
### Goals

1. Make Buttons be capable of Copying to Clipboard
2. OR Inject simple JS into Email HTML

### Background

Really helpful for Marketing Emails,
It has been implemented by Salesforce
[LinkedIn Post](https://www.linkedin.com/posts/rolandeder28674516a_sfmc-salesforcemarketingcloud-salesforce-activity-7082344519857246209-L9S_/)

### Proposal

Not sure exactly how to implement this, but it is a great value add for anyone generating Marketing Mails.

**Comment by AdrKacz** [comment]
@arvind-iyer-2001 did you find a solution?
